### ISSUE TITLE
Default behavior of usageStatistics causes exception on localhost

### ISSUE THREAD

**Issue by jjangga0214** [nhnent/tui.editor#130]
## Version
1.0.5

## Test Environment
ubuntu 16.04, chrome. 64 bits. chrome browser v.65.0.3325.181

## Current Behavior
If I don't configure `usageStatistics: false`, exception is raised. 
By the way, `tui-editor.out-of-the-box.min.js`, appeared on the log below, literally contains all dependencies in desired sequence.  
I use it as __tui.editor__ doesn't provide so-called _all-in-one_ way to use it.

```console
jquery.js:3818 jQuery.Deferred exception: Cannot read property 'toLowerCase' of undefined TypeError: Cannot read property 'toLowerCase' of undefined
    at Object.setRequestHeader (http://localhost:9000/static/bower_components/jquery.js:8988:40)
    at HTMLDocument.<anonymous> (http://localhost:9000/templates/admin/announcement.html:49:17)
    at HTMLDocument.dispatch (http://localhost:9000/static/bower_components/jquery.js:5183:27)
    at HTMLDocument.elemData.handle (http://localhost:9000/static/bower_components/jquery.js:4991:28)
    at Object.trigger (http://localhost:9000/static/bower_components/jquery.js:8249:12)
    at Function.ajax (http://localhost:9000/static/bower_components/jquery.js:9189:24)
    at Function.jQuery.(anonymous function) [as post] (http://localhost:9000/static/bower_components/jquery.js:9355:17)
    at i (http://localhost:9000/static/js/plugin/tui-editor.out-of-the-box.min.js:14436:264863)
    at new e (http://localhost:9000/static/js/plugin/tui-editor.out-of-the-box.min.js:14436:89879)
    at HTMLDocument.<anonymous> (http://localhost:9000/templates/admin/announcement.html:267:24) undefined
jQuery.Deferred.exceptionHook @ jquery.js:3818
process @ jquery.js:3606
setTimeout (async)
(anonymous) @ jquery.js:3640
fire @ jquery.js:3268
fireWith @ jquery.js:3398
fire @ jquery.js:3406
fire @ jquery.js:3268
fireWith @ jquery.js:3398
ready @ jquery.js:3878
completed @ jquery.js:3888
jquery.js:8988 Uncaught TypeError: Cannot read property 'toLowerCase' of undefined
    at Object.setRequestHeader (jquery.js:8988)
    at HTMLDocument.<anonymous> (announcement.html:49)
    at HTMLDocument.dispatch (jquery.js:5183)
    at HTMLDocument.elemData.handle (jquery.js:4991)
    at Object.trigger (jquery.js:8249)
    at Function.ajax (jquery.js:9189)
    at Function.jQuery.(anonymous function) [as post] (http://localhost:9000/static/bower_components/jquery.js:9355:17)
    at i (tui-editor.out-of-the-box.min.js:14436)
    at new e (tui-editor.out-of-the-box.min.js:14436)
    at HTMLDocument.<anonymous> (announcement.html:267)
```

## Tracking
I found that `_jquery2.default.post` causes the exception, which is on the line 24174 of __tui-editor-Editor.js__
```js
function sendHostName() {
  var trackingID = 'UA-115377265-2';
  var hitType = 'event';
  var _location = location,
      hostname = _location.hostname;


  _jquery2.default.post('https://www.google-analytics.com/collect', {
    v: 1,
    t: hitType,
    tid: trackingID,
    cid: hostname,
    dp: hostname,
    dh: hostname
  });
}
```

## Possibility
I tested this on the server a popular grunt plugin `grunt-contrib-watch` provides with livereload connection. Maybe it affects in some way, such as strict outbound rules that may block a request to `https://www.google-analytics.com/collect`, but I haven't researched on it. If that's really due to mis-configuration with grunt, then the caution on the document would be cool. Otherwise, it should be debugged.

**Comment by kyuwoo-choi** [comment]
Yup. It's been reported and resolved in https://github.com/nhnent/tui.editor/pull/122/commits/834506802a5d5331e3182cff0d1ff093215f7a2d

I changed the method from by sending post request to by adding/removing image element.

It'll be released in v1.1.0 soon.

**Comment by kyuwoo-choi** [comment]
https://github.com/nhnent/tui.editor/releases/tag/v1.1.0-a
